bot/seasons/evergreen/bookmark.py

```diff
@@ -0,0 +1,67 @@
+import logging
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+log = logging.getLogger(__name__)
+
+
+class Bookmark(commands.Cog):
+    """A cog that creates personal bookmarks by relaying a message to the user's DMs."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(name="bookmark", aliases=("bm", "pin"))
+    async def bookmark(self, ctx: commands.Context, jump_url: str = None, *args) -> None:
+        """Bookmarks a message."""
+        if jump_url is not None and jump_url != "*":
+            if "discordapp.com/channels" not in jump_url:
+                await ctx.send(
+                    "I can't find the associated message."
+                    " this command supports the following syntax:\n"
+                    "`[command alias] [message url] (bookmark name)`"
+                )
+                return
+        if jump_url is None or jump_url == "*":
+            channel = ctx.channel
+            async for message in channel.history(limit=2):
+                jump_url = message.jump_url
+
+        embed = discord.Embed(
+            title="Your bookmark",
+            description=None,
+            colour=Colours.soft_green
+        )
+        hint = ""
+        for word in args:
+            hint = hint + " " + word

```

Comment:
You can just do
```py
hint = ' '.join(args)
```

Reply:
that `may` cause issue on 
if hint == '' :
    hint = "No hint provided."
Not sure about that.

Reply:
Done anyways , Thanks <3